frontend/src/app/account-activation/account-activation.component.html

```diff
@@ -0,0 +1,2 @@
+<div *ngIf="activationSuccessful === true" class="alert alert-success" role="alert">Pomyślnie aktywowano użytkownika</div>
```

Comment:
`activationSuccessful` is defined as boolean in TypeScript, is it really necessary to check it using `...=== true` here?

BTW why don't use  `else` here? see: https://angular.io/api/common/NgIf#showing-an-alternative-template-using-else

Reply:
@jacol12345, do You remember why we didn't use `else` here? I think there was some explanation, but cannot recall it.